dcap/types/src/certification_data.rs

```diff
@@ -59,12 +97,20 @@ impl<'a> CertificationData<'a> {
     ///
     /// `Certification Data` member from Table 9 of
     /// <https://download.01.org/intel-sgx/latest/dcap-latest/linux/docs/Intel_SGX_ECDSA_QuoteLibReference_DCAP_API.pdf>.
-    pub fn data(&self) -> &[u8] {
-        self.data
+    pub fn raw_data(&self) -> &[u8] {
```

Comment:
I left this and renamed to `raw_data` to better communicate it's just the buffer.
Not sure this is needed, but it made the testing more convenient up in quote3.rs